src/pages/office/cat-be/office.cat-be.page.html

```diff
@@ -37,11 +37,11 @@ <h4>Test summary</h4>
 
           <activity-code id="activity-code-card" [formGroup]="form"
                          [activityCodeModel]="pageState.activityCode$ | async" [activityCodeOptions]="activityCodeOptions"
-                         (activityCodeChange)="activityCodeChanged($event)" [disabled]="!isTerminated() || !isDelegated"></activity-code>
+                         (activityCodeChange)="activityCodeChanged($event)" [disabled]="!isDelegated"></activity-code>
```

Comment:
will the [disabled] check mean that if the testoutcome is a pass or a fail to begin with it will be changeable? on the pass story it needs to be disabled, imagine the same would be for the fail journey

Reply:
This basically defaults it back to the way it works now for non delegated tests (below). Activity codes < 4 are disabled within the  `activity-code `component
`[disabled]="true"`

Reply:
completely understandable, i couldn't see the logic regarding that in the pr but that makes sense, everything else looks good, will approve on the basis matts comment gets addressed if its an issue